src/common/preferences/preferences.html

```diff
@@ -35,53 +35,51 @@
 				<div class="pane-image"><img src="../images/prefs-general.png" alt=""/></div>
 				<div class="pane-label">General</div>
 			</div>
+			<div id="pane-proxies" class="pane" style="display: none">
+				<div class="pane-image"><img src="../images/prefs-proxies.png" alt=""/></div>
+				<div class="pane-label">Proxies</div>
+			</div>
 			<div id="pane-advanced" class="pane">
 				<div class="pane-image"><img src="../images/prefs-advanced.png" alt=""/></div>
 				<div class="pane-label">Advanced</div>
 			</div>
 		</div>
 		<div id="content-general" class="content">
 			<div class="group">
-				<div class="group-title">Zotero Standalone</div>
+				<div class="group-title">Zotero Status</div>
 				<div class="group-content">
-					<p>Zotero Standalone is currently <span id="general-span-standalone-status"></span>.</p>
+					<p>Zotero is currently <span id="general-span-standalone-status"></span>.</p>
 					<p><input type="button" value="Update Status" id="general-button-update-standalone-status"/></p>
 				</div>
 			</div>
 			<div class="group">
-				<div class="group-title">Save via Server</div>
+				<div class="group-title">Save to Zotero.org</div>
```

Comment:
> They don't? What do people use then?

They [definitely do not](https://www.theguardian.com/technology/blog/2010/feb/11/facebook-readwriteweb). They use Google. Which doesn't mean that they wouldn't necessarily understand "Save to Zotero.org", but I do think it's a little technical.

I wasn't suggesting "Save to Server", to be clear — that should definitely be replaced.